rtv/content.py

```diff
@@ -155,9 +169,13 @@ def get(self, index, n_cols=70):
 
         elif index == -1:
             data = self._submission_data
-            data['split_title'] = textwrap.wrap(data['title'], width=n_cols-2)
-            data['split_text'] = wrap_text(data['text'], width=n_cols-2)
-            data['n_rows'] = len(data['split_title'])+len(data['split_text'])+5
+            data['split_title'] = textwrap.wrap(
+                data['title'],
+                width=n_cols -
+                2)
+            data['split_text'] = wrap_text(data['text'], width=n_cols - 2)
+            data['n_rows'] = len(
+                data['split_title']) + len(data['split_text']) + 5
```

Comment:
This one is kind of strange, I think it implies that the whole second line is wrapped by len(). I think it might be clearer to keep on a single line.